documentation/api/io.strimzi.api.kafka.model.kafka.listener.KafkaListenerAuthenticationCustom.adoc

```diff
@@ -40,7 +40,37 @@ A protocol map is generated that uses the `sasl` and `tls` values to determine w
 Secrets are mounted to `/opt/kafka/custom-authn-secrets/custom-listener-<listener_name>-<port>/<secret_name>` in the Kafka broker nodes' containers.
 For example, the mounted secret (`example`) in the example configuration would be located at `/opt/kafka/custom-authn-secrets/custom-listener-oauth-bespoke-9093/example`.
 
+= Configuring customized TLS Client Authentication
+
+You can also use the `custom` authentication to also configure customized TLS Client Authentication
+That allows you to configure options not allowed with `type: tls` authentication.
+For example a custom truststore with multiple trusted CAs or options such as `ssl.principal.mapping.rules`.
+
+.Example custom TLS Client Authentication configuration
+[source,yaml,subs="attributes+"]
+----
+spec:
+  kafka:
+    listeners:
+      - name: tls
+        port: 9093
+        tls: true
+        type: internal
+        authentication:
+          type: custom
+          sasl: false
+          listenerConfig:
+            ssl.client.auth: required
+            ssl.principal.mapping.rules: RULE:^CN=(.*?),(.*)$/$1@my-cluster.com/
+            ssl.truststore.location: /opt/kafka/custom-authn-secrets/custom-listener-tls-9093/custom-truststore/ca.crt
```

Comment:
this is an actual path on the broker pod where the CA cert is mounted, but how the user knows about this path to configure this field? I mean, is it documented somewhere? I couldn't find it.

Reply:
The oath where the secrets are mounted is listed in the docs: https://strimzi.io/docs/operators/latest/full/configuring.html#type-KafkaListenerAuthenticationCustom-schema-reference

Reply:
Ok thanks, I missed that. I think it could be useful to have a reference to this doc in the example above.

Reply:
The example above is in the same section where the schema is.

Reply:
ok then, thanks.